MapleServer2/Packets/FieldPacket.cs

```diff
@@ -124,7 +123,7 @@ public static class FieldPacket {
                 pWriter.WriteInt();
                 pWriter.WriteShort();
             } else {
-                pWriter.WriteInt();
+                //pWriter.WriteInt(); commented out to remove warning
```

Comment:
Are we currently aware as to why this is throwing a warning? I typically don't recommend commenting stuff out just to hide warnings because we will forget about it.

Reply:
Unreachable code from a hardcoded if statement because the packet isn't entirely decrypted.

Reply:
This one is because it never reach the _else_ as it said the warning.